### PR TITLE
Replace readme

### PR DESCRIPTION
**Pull request by yenienserrano**
### Description
Update the `README` with the project information.

### Issues Resolved
- #13 

### Check List
- [ ] New functionality includes testing
- [ ] New functionality has been documented
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,20 +36,17 @@ See [developer guide](DEVELOPER_GUIDE.md) and [how to contribute to this project
...
-
-## Code of Conduct
-
-This project has adopted the [Amazon Open Source Code of Conduct](CODE_OF_CONDUCT.md). For more information see the [Code of Conduct FAQ](https://aws.github.io/code-of-conduct-faq), or contact [opensource-codeofconduct@amazon.com](mailto:opensource-codeofconduct@amazon.com) with any additional questions or comments.
+For more information, see [project website](https://wazuh.com/) and [documentation](https://documentation.wazuh.com/current/index.html). If you need help and are unsure where to open an [issue](https://github.com/wazuh/wazuh-security-dashboards-plugin/issues).
 
 ## Security
 
-If you discover a potential security issue in this project we ask that you notify AWS/Amazon Security via our [vulnerability reporting page](http://aws.amazon.com/security/vulnerability-reporting/). Please do **not** create a public GitHub issue.
+If you discover a potential security issue in this project we ask that you notify AWS/Amazon Security via our [vulnerability reporting page](https://github.com/wazuh/wazuh-kibana-app/issues/new/choose). Please do **not** create a public GitHub issue.
```
Comment (AlexRuiz7) [inline README.md:43]: Leads to the kibana-app repo

**Review by AlexRuiz7 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (yenienserrano) [reply README.md]: changed

**Review by AlexRuiz7 — APPROVED** [review]
LGTM! Great work!

**Review by AlexRuiz7 — APPROVED** [review]
LGTM! Great work!

**Pull request merged**